sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -132,16 +132,35 @@ static inline std::string codeToString(cl_int code) {
 #define __SYCL_REPORT_OCL_ERR_TO_EXC_BASE(code)                                \
   __SYCL_REPORT_OCL_ERR_TO_STREAM(code)
 #endif
+// SYCL 2020 exceptions
+#define __SYCL_REPORT_OCL_ERR_TO_EXC_VIA_ERRC(expr, errc)                      \
```

Comment:
```suggestion
#define __SYCL_REPORT_ERR_TO_EXC_VIA_ERRC(expr, errc)                      \
```

Let's stop tying implementation to OpenCL

Reply:
good call. Thanks. Done.